specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/examples/ExternalBillingAccountsDimensions.json

```diff
@@ -0,0 +1,45 @@
+{
+  "parameters": {
+    "api-version": "2022-10-01",
+    "externalCloudProviderId": "100",
+    "externalCloudProviderType": "externalBillingAccounts"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "providers/Microsoft.CostManagement/externalBillingAccounts/100/dimensions_ResourceType_2019-12-01_2019-12-31",
+            "name": "dimensions_ResourceType_2019-12-01_2019-12-31",
+            "type": "microsoft.consumption/dimensions",
+            "properties": {
+              "data": [],
```

Comment:
Are these empty examples of data: [] good ones for understanding how to completely parse the response?

Reply:
@absyed please comment or help update this example.

Reply:
Updated example.